src/ciflow-bot.ts

```diff
@@ -162,17 +161,22 @@ export class CIFlowBot {
         name: label
       });
     }
-    await this.ctx.github.issues.addLabels({
-      owner: this.ctx.payload.repository.owner.login,
-      repo: this.ctx.payload.repository.name,
-      issue_number: this.pr_number,
-      labels: labelsToAdd
-    });
+    if (labelsToAdd.length > 0) {
```

Comment:
when no new labels to add, just skip this api request

Reply:
nit: can you inline these explanation into the code itself as comments?